apps/astarte_trigger_engine/test/astarte_trigger_engine_web_test.exs

```diff
@@ -0,0 +1,56 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.TriggerEngineWebTest do
+  use Astarte.Cases.Database, async: true
+  use ExUnitProperties
+
+  import Astarte.Helpers.Plug
+  import Astarte.Helpers.Telemetry
+
+  test "/health returns ok if astarte is working correctly" do
+    conn = get("/health")
+
+    # FIXME: use 200 when this actually works
+    assert conn.status == 503
+  end
+
+  test "/metrics return prometheus metrics" do
+    conn = get("/metrics")
+
+    telemetry = TelemetryMetricsPrometheus.Core.scrape()
+    resp_headers = Map.new(conn.resp_headers)
+
+    assert same_telemetry?(conn.resp_body, telemetry)
```

Comment:
As agreed upon IRL, checking that `TelemetryMetricsPrometheus.Core.scrape()` is called is an implementation detail that can be left out at this level. This is mainly due to the fact that metrics  handling is a well-known component and that increase in coverage is marginal